src/picom.c

```diff
@@ -627,15 +658,69 @@ static void handle_root_flags(session_t *ps) {
 	}
 }
 
-static struct managed_win *paint_preprocess(session_t *ps, bool *fade_running) {
+static struct managed_win *paint_preprocess(session_t *ps, bool *fade_running, bool *animation_running) {
```

Comment:
Can fading be merged with this animation code?

Reply:
Pretty sure it can!

Reply:
Yes!